### DISCUSSION TITLE
Unable to complete signup

### DISCUSSION THREAD

**Discussion by TheAwesomeTechGirl** [vercel/community#3445] [Help] [answered]
### Summary

Good day vercel 
My name is Tuke Glory Nwakego
i've been trying to sign up with my github account and it has been telling me the same reply that :There is already an account associated with your Github email address .please login with your email and password then link your github account .
I tried login with my email and i got this response:we are unable to complete your signup .And because of this i have tried creating a different email to login ,i even created a new github account to login but still same response .sorry to have tried so much on different means i should have sent a mail but i didn't know what email address to send it to.but i found this on the github page.
This is the [Email:tukeglory98@gmail.com](mailto:Email%3Atukeglory98@gmail.com) and the username :TheAwesomeTechGirl,I would be settling with this for the account.I hope to hear from you soon and i'm really sorry for trying out multiple times.

### Example

no response

### Steps to Reproduce

sign up with github 
was redirected to signup with email
couldn't complete the sign up process

**Comment by amyegan** [comment, marked as answer]
Hey @TheAwesomeTechGirl. If you already sent an email to [registration@vercel.com](mailto:registration@vercel.com), they will be in touch with you soon to sort out this problem as quickly as possible.

**Reply by TheAwesomeTechGirl** [reply]
Noted 
thank you

**Comment by truongagile** [comment]
in my case i got  [error too](https://prnt.sc/ADv59YWth1Qj) and sent to [registration@vercel.com](mailto:registration@vercel.com) and still not get reply!

**Reply by ghost** [reply]
Hello!
Did you fix that?

**Comment by ghost** [comment]
Hello
Did you fix that?
